markdown/ranger/ranger-integration-config.html.md.erb

```diff
@@ -84,19 +105,28 @@ The following procedures describe each configuration activity.
     gpadmin@master$ hawq stop cluster --reload
     ```
 
-7. To validate connectivity between Ranger and HAWQ, access the Ranger Admin UI in Ambari, click the edit icon associated with the `hawq` service definition. Ensure that the Active Status is set to Enabled, and click the **Test Connection** button. You should receive a message that Ranger connected succesfully.  If it fails to connect, edit your HAWQ connectivity properties directly in the Ranger Admin UI and re-test the connection.
+7.  When setup is complete, use the fully-qualified domain name to log into the Ambari server. Use the Ranger link in the left nav to bring up the Ranger Summary pane in the HAWQ Ambari interface. Use the Quick Links to access Ranger. This link will take you to the Ranger Login interface. 
```

Comment:
A direct URL would be good if that can be generalized.  Otherwise this should probably be broken into multiple steps (there's a lot of UI control going on).  Ideally I don't want us to go too far into navigating the ranger UI itself, except as it relates to our plugl-in.